samples/DS3232RTC_NTP_Setter/app/application.cpp

```diff
@@ -38,29 +37,21 @@ void gotIP(IPAddress ip, IPAddress netmask, IPAddress gateway)
 	ntpClient.requestTime();
 }
 
-void connectFail(String ssid, uint8_t ssid_len, uint8_t bssid[6], uint8_t reason)
-{
-	debugf("I'm NOT CONNECTED!");
-}
-
 void init()
 {
 	Serial.begin(SERIAL_BAUD_RATE);
-	Serial.systemDebugOutput(true); // Allow debug print to serial
 	Serial.println("Sming DSRTC_NTP_SETTER started!");
-//Uncomment proper line, or set right pins by hand
 	Wire.pins(0, 2); //Change to your SCL - 0,SDA - 2 GPIO pin number
```

Comment:
I think we shoudn't use Wire.pins(0, 2) by default, it is already default value, so better to comment out Wire.pins occurrences with comments how to set non-default scl/sda pins.